x/relationships/keeper/keeper_test.go

```diff
@@ -0,0 +1,120 @@
+package keeper_test
+
+import (
+	"fmt"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/desmos-labs/desmos/x/relationships/types"
+)
+
+func (suite *KeeperTestSuite) TestKeeper_StoreRelationship() {
+	tests := []struct {
+		name                string
+		storedRelationships []sdk.AccAddress
+		user                sdk.AccAddress
+		receiver            sdk.AccAddress
+		expErr              error
+	}{
+		{
+			name:                "already existent relationship returns error",
+			storedRelationships: []sdk.AccAddress{suite.testData.otherUser},
+			user:                suite.testData.user,
+			receiver:            suite.testData.otherUser,
+			expErr:              fmt.Errorf("relationship already exists with %s", suite.testData.otherUser),
+		},
+		{
+			name:                "relationship added correctly",
+			storedRelationships: nil,
+			user:                suite.testData.user,
+			receiver:            suite.testData.otherUser,
+			expErr:              nil,
+		},
+	}
+
+	for _, test := range tests {
+		suite.SetupTest()
+		suite.Run(test.name, func() {
+			if test.storedRelationships != nil {
+				store := suite.ctx.KVStore(suite.keeper.StoreKey)
+				store.Set(types.RelationshipsStoreKey(test.user), suite.keeper.Cdc.MustMarshalBinaryBare(&test.storedRelationships))
+			}
+			err := suite.keeper.StoreRelationship(suite.ctx, test.user, test.receiver)
+			suite.Equal(test.expErr, err)
+		})
+	}
+}
+
+func (suite *KeeperTestSuite) TestKeeper_GetUserRelationshipsMap() {
+	relationshipsMap := map[string][]sdk.AccAddress{
+		suite.testData.user.String():      {suite.testData.otherUser},
+		suite.testData.otherUser.String(): {suite.testData.user},
+	}
+
+	_ = suite.keeper.StoreRelationship(suite.ctx, suite.testData.user, suite.testData.otherUser)
+	_ = suite.keeper.StoreRelationship(suite.ctx, suite.testData.otherUser, suite.testData.user)
+
+	actualIDsMap := suite.keeper.GetUsersRelationships(suite.ctx)
+	suite.Equal(relationshipsMap, actualIDsMap)
+}
+
+func (suite *KeeperTestSuite) TestKeeper_GetUserRelationships() {
```

Comment:
Can we add a test case where we try to read the relationships with an empty slice?